sdk/storage/azure-storage-common/test/test_base.cpp

```diff
@@ -141,7 +141,7 @@ namespace Azure { namespace Storage { namespace Test {
     return connectionString;
   }
 
-  std::string AppendQueryParameters(Azure::Core::Url const& url, std::string const& queryParameters)
+  std::string AppendQueryParameters(const Azure::Core::Url& url, const std::string& queryParameters)
```

Comment:
@Jinming-Hu - we are not consistent within storage/core about the ordering of `const`. Should we decide on and settle on one particular ordering and update all instances?

In many places within core/keyvault, we use `type_name const&`.

I saw storage using `const type_name&` in most places, except for when it is overriding base classes from core.
https://github.com/Azure/azure-sdk-for-cpp/blob/4db9a563e75bd7f67ffd3f2a895577ecda122ef1/sdk/storage/azure-storage-common/inc/azure/storage/common/reliable_stream.hpp#L55

https://github.com/Azure/azure-sdk-for-cpp/blob/4db9a563e75bd7f67ffd3f2a895577ecda122ef1/sdk/storage/azure-storage-common/inc/azure/storage/common/shared_key_policy.hpp#L29-L32

Reply:
> we are not consistent within storage/core about the ordering of const. Should we decide on and settle on one particular ordering and update all instances?

Yeah, I know. But it's better if we can keep consistent in each sdk than not even consistent within one sdk, isn't it?

Reply:
> Yeah, I know. But it's better if we can keep consistent in each sdk than not even consistent within one sdk, isn't it?

Yes :)